src/helix/client/client_ext.rs

```diff
@@ -998,6 +998,24 @@ impl<'client, C: crate::HttpClient + Sync + 'client> HelixClient<'client, C> {
         make_stream(req, token, self, |broadcasts| broadcasts.segments.into())
     }
 
+    /// Gets the broadcaster’s streaming schedule as an [iCalendar](https://datatracker.ietf.org/doc/html/rfc5545).
+    pub async fn get_channel_icalendar<'b, T>(
+        &'client self,
+        broadcaster_id: impl types::IntoCow<'b, types::UserIdRef> + Send + 'b,
+        token: &T,
```

Comment:
I don't know either. So, I moved it to https://github.com/twitch-rs/twitch_api/pull/463. My initial thought was to use an empty token (i.e. one that always returns `""`), but that wouldn't work well. Maybe this could be an endpoint that's only defined on `ClientExt` where we use reqwest directly? Either way, let's do this in the other PR.